src/client/components/ItemPage/ItemPage.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useRef } from 'react';
+import Chart from 'chart.js';
+import PropTypes from 'prop-types';
+
+import { getItemById } from '../../services/api';
+
+const generateChart = (ref, data) => (
+  new Chart(ref, {
+    type: 'line',
+    data: {
+      labels: data.history.map(({ createdAt }) => new Date(createdAt).toLocaleDateString()),
+      datasets: [{ label: 'History', data: data.history.map(({ price }) => price) }],
+    },
+  })
+);
+
+const ItemPage = ({ match: { params: { id } } }) => {
+  const chartRef = useRef(null);
+
+  useEffect(() => {
+    const handleItemFetch = async () => {
+      try {
+        const { data } = await getItemById(id);
+
+        if (data.history.length) {
+          const myChartRef = chartRef.current.getContext('2d');
+          generateChart(myChartRef, data);
+        }
+      } catch (error) {
+        // TODO: handle error
+      }
+    };
+
+    handleItemFetch();
+  }, []);
+
+  return (
+    <div>
+      <canvas id="chard" ref={chartRef} />
```

Comment:
chart?

Reply:
Maybe, we can somehow export History  chart into standalone component to later reuse it on other pages?

Reply:
maybe but not in this PR